src/misc/sw-template.js

```diff
@@ -237,12 +238,26 @@ function WebpackServiceWorker(params) {
   }
 
   self.addEventListener('fetch', (event) => {
-    const url = new URL(event.request.url);
-    url.search = '';
-    const urlString = url.toString();
+    // Handle only GET requests
+    if (event.request.method !== 'GET') {
```

Comment:
Too early exit. This doesn't go through Firefox's fix for cross-origin requests.